packages/Php74/tests/Rector/Property/TypedPropertyRector/Fixture/parent_of_parent_has_untyped_property.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Rector\Php74\Tests\Rector\Property\TypedPropertyRector\Fixture;
+
+abstract class Middle extends \SomeNamespace\SomeParent
+{
+}
+
+final class Child2 extends Middle
+{
+    /**
+     * @var string
+     */
+    protected $name = 'child';
+
+    /**
+     * @var string
+     */
+    protected string $typedName = 'child';
+}
+
+?>
+-----
```

Comment:
If the file should not be changed at all, you can remove all content from here to the end of the file.

Reply:
Removed.